worldengine/draw.py

```diff
@@ -209,26 +393,93 @@ def draw_riversmap(world, target):
 
 
 def draw_grayscale_heightmap(world, target):
-    e = world.elevation['data']
+    c = get_normalized_elevation_mask(world)
```

Comment:
Ah, now I understand why you mimicked this behavior. Well, I guess it's fine then.